src/index.ts

```diff
@@ -542,24 +487,18 @@ class ForkTsCheckerWebpackPlugin {
       }
     );
 
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     this.serviceRpc = new RpcProvider(message => this.service!.send(message));
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     this.service.on('message', message => this.serviceRpc!.dispatch(message));
 
     const forkTsCheckerHooks = ForkTsCheckerWebpackPlugin.getCompilerHooks(
       this.compiler
     );
-    forkTsCheckerHooks.serviceStart.call(
-      this.tsconfigPath,
-      this.tslintPath,
-      this.memoryLimit
-    );
+    forkTsCheckerHooks.serviceStart.call(this.tsconfigPath, this.memoryLimit);
 
     if (!this.silent && this.logger) {
-      this.logger.info(
-        'Starting type checking' +
-          (this.tslint ? ' and linting' : '') +
-          ' service...'
-      );
+      this.logger.info('Starting type checking service...');
```

Comment:
Do we want a comment here to indicate when linting is happening too?

Reply:
I would leave it as it is in this PR as I plan to refactor plugin logs anyway :)